### PR TITLE
chore: update maintainers file

### LINKED ISSUE #620: Update the project maintainers to accurately reflect the current state of contributors.
The active contributors has substantially changed over the past several months. We need to update the maintainers file to better reflect the the current state of contributors.

The following maintainers will be added:

1. Ryan Koch

The following maintainers will be moved to emeritus status:

1. Karim Stekelenburg
2. Sam Curren
3. Timo Glastra

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Proposed changes to the project maintainers.

Fixes #620

# Related Issues

Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: MAINTAINERS.md
Comment (ryankoch13) [inline MAINTAINERS.md]: I would rather use my other github id here - @ryankoch13

**Comment by jleach** [comment]
Reviewed on Bifold UG call. Will merge once a fix is in for Ryan's Github ID.

**Review by ryankoch13 — COMMENTED** [review]
as soon as it's merged I can approve it :P

**Review by cvarjao — APPROVED** [review]

**Review by amanji — APPROVED** [review]

**Review by ryankoch13 — COMMENTED** [review]
~LGTM!

**Pull request merged**
